plugins/gimimprove

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/TheSimpleBlue/gimimprove.git
+commit=255e0aaf55143d604b9a2da0a76be6eea4e97b48
```

Comment:
This needs a warning as it has a default configuration which submits requests to non-verified servers.
```suggestion
commit=255e0aaf55143d604b9a2da0a76be6eea4e97b48
warning=This plugin submits your IP address to a server not controlled or verified by the RuneLite developers.
```